_targets/build.project.armv7a9-zynq7000

```diff
@@ -86,17 +95,50 @@ b_build_target() {
 
 b_flash_image() {
 	local flash_path="${PREFIX_BOOT}/flash-${TARGET}.bin"
-
 	rm -f "${flash_path}"
-	dd if="${PREFIX_BOOT}/phoenix-${TARGET}.disk" of="${flash_path}" bs=4M 2>/dev/null
+
+	dd if="${PREFIX_BOOT}/phoenix-${TARGET}.disk" of="${flash_path}" bs=4M conv=notrunc 2>/dev/null
+	dd if="${PREFIX_BOOT}/phoenix-${TARGET}.jffs2" of="${flash_path}" bs=4096 seek=$((FS_OFFS/4096)) conv=notrunc 2>/dev/null
+
 	truncate -s 16M "${flash_path}" 2>/dev/null
 }
 
 
+b_build_jffs2() {
+	# !!! NOTE:
+	#         For non-standard NOR flash memories, erase_sz & page_sz should be changed..
+	local erase_sz=$((0x10000))
+	local page_sz=$((0x100))
+
+	# FIXME: does work only on macOS, for now hackish solution based on extending PATH
+	mtd-utils/build.sh
+	PATH="$(pwd)/_build/host-pc/prog/:$PATH"
```

Comment:
```suggestion
	OLD_PATH="$PATH"
	PATH="$(pwd)/_build/host-pc/prog/:$PATH"
	...
	PATH="$OLD_PATH"
```
For safety, you should keep current `PATH` and recreate it at the end of the function.